src/components/CollapsedComponent.jsx

```diff
@@ -0,0 +1,46 @@
+import {faChevronUp} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import React from 'react';
+import PropTypes from 'prop-types';
+import partial from 'lodash-es/partial';
+import classnames from 'classnames';
+
+export default function CollapsedComponent({
+  component,
+  isRightJustified = true,
+  onComponentUnhide,
+  projectKey,
+  text,
+}) {
+  return (
+    <div
+      className="editors__collapsed-editor"
+      onClick={partial(
+        onComponentUnhide,
+        projectKey,
+        `editor.${component}`,
```

Comment:
I think the `editor.` bit should be part of what the outer component is passing in as the `component` prop?